frontend/src/components/coverage/enrollment/RegistrationConfirmation.vue

```diff
@@ -10,6 +10,7 @@
         <AppOutput label="Name" :value="fullName" />
       </AppCol>
     </AppRow>
+    <AppButton @click="goToAddAnotherPermitHolder()" mode="primary" type="button">Add Another Permit Holder</AppButton>
```

Comment:
Should the text indicate if the user will add Permit Holder with or without PHN as could navigate to either.